io/file.go

```diff
@@ -233,6 +233,7 @@ func GetDiskStat(path string) (diskStat types.DiskStat, err error) {
 		DiskID:           fsidFormatted,
 		Path:             path,
 		Type:             usage.Fstype,
+		Driver:           "",
```

Comment:
Why it's empty instead of retrieving from the disk info.

Reply:
Addressed. Thank you.